tools/GATK-ApplyRecalibration.cwl

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+
+
+hints:
+- $import: envvar-global.yml
+- $import: GATK-docker.yml
+
+inputs:
+  multithreading_nt:
+    type: int
+    default: 4
+    inputBinding:
+      position: 12
+      prefix: --nt
+    doc: multithreading option
+
+  reference:
+    type: File
+    inputBinding:
+      position: 6
+      prefix: -R
+    doc: reference genome
```

Comment:
What format is this file? You can search the EDAM Ontology at http://bioportal.bioontology.org/ontologies/EDAM?p=classes